### PR TITLE
ci: appveyor

### PR DESCRIPTION
**Pull request by evilebottnawi**
/cc @vjeux looks we don't have permissions to add project https://ci.appveyor.com/project/prettier/plugin-php

### CODE REVIEW

**Comment by evilebottnawi** [comment]
Looks we have bug with pragma on `windows` :confused:

**Comment by vjeux** [comment]
Do you know what kind of permission it requires, you are admin on the project:

image

When I click on that link I see

image

but I don't have an appveyor account

**Comment by vjeux** [comment]
I was able to create https://ci.appveyor.com/project/vjeux/plugin-php still trying to get it under prettier account...

**Comment by vjeux** [comment]
@evilebottnawi I added you as admin of the prettier org itself. Please let me know if you can make it work. I don't understand how to do it from their UI :x

**Comment by evilebottnawi** [comment]
@vjeux still not enough rights/permissions :disappointed: For other projects i have  "+ New project" in the top left side. You can change account in upper top left side - just click on account/organization name. Maybe need add me here https://ci.appveyor.com/account/prettier/team

**Comment by vjeux** [comment]
What's your appveyor email?

**Comment by evilebottnawi** [comment]
@vjeux Usually this is used to avoid addition and removal (after setup hook appveyor should add permissions to all administrators of repo) https://ci.appveyor.com/account/prettier/github-teams

**Comment by vjeux** [comment]
Can you try again now?

**Comment by evilebottnawi** [comment]
@vjeux no, looks you add me access to you account, not `prettier` org

**Comment by vjeux** [comment]
Grrrrr!! Sorry let me try again in 10min when I’m at my desk :(

**Comment by vjeux** [comment]
When I go to the prettier url, unfortunately it assumes that it's for vjeux... I don't know how to change it to use prettier.

image

**Comment by evilebottnawi** [comment]
@vjeux very strange, maybe we should ask someone for help

**Comment by vjeux** [comment]
Let me delete it and then you can try to add it to the org. You are admin of the prettier org on github so you should be able to do it.

**Comment by vjeux** [comment]
Done

**Comment by evilebottnawi** [comment]
@vjeux Can't add :disappointed: What the hell with approve :confused:

**Comment by evilebottnawi** [comment]
@vjeux Can you have access to `https://ci.appveyor.com/account/prettier/team`? Maybe you can add me as Administrator (`sheo13666q@gmail.com`) and i try to find how it should be setup

**Comment by vjeux** [comment]
You are already an admin when I look at this page...

image

**Comment by evilebottnawi** [comment]
@vjeux you should switch account to `prettier`, you screenshot show what you add me as Administrator for you account (`vjeux`), not `prettier`. Just click on account name on navbar and switch to `prettier`

**Comment by vjeux** [comment]
I can't do that...

image

**Comment by evilebottnawi** [comment]
@vjeux it seems that someone else has rights, who set up `appveyor` for `prettier`? Maybe he/she/it forgot setup github org hook for access

**Comment by vjeux** [comment]
I have no idea who that would be...

**Comment by evilebottnawi** [comment]
/cc @azz 
Based on history https://github.com/prettier/prettier/commits/master/.appveyor.yml

**Comment by evilebottnawi** [comment]
Awesome, windows tests passed

**Comment by ikatyang** [comment]
It's me. (I forgot to mention here that the discussion has been moved to prettier/prettier#5181.)
